src/containers/StructurePage/resourceComponents/Resource.tsx

```diff
@@ -143,6 +155,25 @@ const Resource = ({
       </StyledLink>
     );
 
+  const WrongTypeError = () => {
```

Comment:
Usikker på om denne også bør skjules for vanlige brukere av ed. Eller kanskje det er lurt at dei får denne visninga, slik at dei kan seie ifra til andre om at dette bør endres.

Reply:
Det kan jo være med å bevisstgjøre brukerene. Tenker vi kan la det stå inntil vi får beskjed om noko anna.